.github/workflows/test.yml

```diff
@@ -0,0 +1,39 @@
+name: Tests
+on: pull_request
+
+jobs:
+  rubocop:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+
+    - name: Set up Ruby
+      uses: ruby/setup-ruby@v1.49.0
```

Comment:
You should change this to use `ruby/setup-ruby@v1` now (similarly, everwhere else). You also don't need the `bundle install` step after it in either place. This action does the whole shazam and makes you breakfast.

Reply:
Done!